runtimes/peregrine/src/lib.rs

```diff
@@ -738,6 +845,9 @@ construct_runtime! {
 		// Parachains pallets. Start indices at 80 to leave room.
 		ParachainSystem: cumulus_pallet_parachain_system = 80,
 		ParachainInfo: parachain_info::{Pallet, Storage, Config} = 81,
+
+		// Proxy pallet.
+		Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 90,
```

Comment:
Same as below by @weichweich: I would suggest 43 because it is a utility pallet.